### DISCUSSION TITLE
Issues setting up a developer subscription to test the samples in the Teams Toolkit

### DISCUSSION THREAD

**Discussion by mookmac** [microsoft/hack-together-teams#9] [Q&A]
I've installed the latest Teams Toolkit into VS Code and opened up the TeamsChef sample. I'm going through the instructions and want to debug the app. I clicked the option to setup a test tenant and managed to sucessfully register for the Microsoft 365 Developer Program. From the Dashboard I've created a subscription using the "instant" option. The subscription has been created and I can see it in my Dashboard, when I then click "Go to subscription" it sends me to a login page with the new admin username and autogenerated domain pre-populated. This seems to be all fine, however when I then click the Next button I'm given an error message saying the username isn't recognised. Is there something I'm missing with this process? Is there a Microsoft support number/chat that I need to speak to about this?

**Comment by therealjohn** [comment]
Weird, you should be able to login using that new admin email address and password. It typically looks like `username@on.microsoft.com` or maybe `@xyz.microsoft.com`. 

Are you able to login to Teams Toolkit in VS Code?

**Reply by mookmac** [reply]
Yep it's definitely the right username - it's shown on the dashboard when I log into the m365 developer program. 

Unfortunately the same issue if I try to log in via VS Code as well. 

It doesn't seem like I've done anything wrong so I'm going to wait and try again tomorrow and hope it magically resolves itself.

**Comment by Obelixx** [comment]
Hello all,

I have the same issue with new Microsoft 365 Developer Program subscription with instant sandbox option. The login does not exist on Microsoft login page (https://login.microsoftonline.com) redirected from "Go to subscription" link that looks like: https://www.office.com/login?login_hint=MyNewUser@MyNewTenant.onmicrosoft.com

**Reply by waldekmastykarz** [reply]
Are you getting any error when trying to login?

**Reply by Obelixx** [reply]
Something in a spirit of: "This username is not correct. Check if your input is correct. Or connect with your administrator."

Error Code: 50058
Request Id: efc763d2-aa6e-48b1-88ae-857602f4a201
Correlation Id: 84e78343-730c-486c-853c-60dabfe42d9c
Timestamp: 2023-06-02T07:50:16.370Z

**Reply by waldekmastykarz** [reply]
Is the error still there? Just want to check in case provisioning the tenant got delayed for some reason.

**Reply by Obelixx** [reply]
Yes, the error is still there.

**Reply by waldekmastykarz** [reply]
Sorry to hear that. Have you tried creating another tenant?

**Reply by Obelixx** [reply]
No, I emailed @therealjohn my tenant info from my development program registration email and waiting.
I'm afraid to click "Delete profile" because (as the documentation says) I'll have to wait 60 days to create a new one.

**Reply by Obelixx** [reply]
The issue is resolved now. I got notification that my subscription is pre-expired and I created a new one successfully. Logged in successfully also. ✔️  
Thanks! 🥂

**Comment by therealjohn** [comment]
We're looking into what's going on but will need a Tenant ID and admin email/username that are you trying. Can you email me with this info? johmil@microsoft.com

**Reply by mookmac** [reply]
Thanks John, I also still have the same issue. Will email through details of my tenant now.

**Reply by Obelixx** [reply]
I also emailed my tenant info.
Thanks for your help!

**Reply by pallav243** [reply]
Was this issue resolved ? I am also facing the same issue.  I'll email my credentials to johmil@microsoft.com as well

**Comment by pnadal** [comment]
Hi there, I have the same issue. I've just subscribed to the developer program with my personal account. I has created an admin email address, but this errors appears when trying to login with the new address: This username may be incorrect. Make sure you typed it correctly. Otherwise, contact your admin.

**Comment by therealjohn** [comment]
The issue is fixed now. If you got this error, you’ll have to create a new tenant. I know that’s very disruptive. If you email me your info I can help get support to reset existing tenants instead of a new one is not an option. Thanks for your understanding.
